src/test/java/games/strategy/engine/framework/map/download/DownloadFileParserTest.java

```diff
@@ -11,6 +11,9 @@
 import games.strategy.io.IoUtils;
 
 public class DownloadFileParserTest {
+  private enum ValueType {
+    MAP, MAP_TOOL, MAP_SKIN, MAP_MOD
+  }
```

Comment:
It looks like `DownloadFileParser#ValueType` wasn't used because `DownloadFileParser` uses `DownloadFileDescription#DownloadType` instead.  I think it would be better if, instead of moving this enum here, we replace its usage below with `DownloadFileDescription#DownloadType`.